src/shared/gitlab-client.ts

```diff
@@ -289,13 +355,16 @@ export class GitlabClient {
     return this.fetchJson<Group[]>(`groups`, true);
   }
 
-  public async getProject(projectId: number, userIdOrName?: number | string) {
+  public async getProject(projectId: number, userIdOrName?: number | string) {
     let project: Project;
     if (userIdOrName) {
       const sudo = {
         sudo: userIdOrName,
       };
-      project = await this.fetchJson<Project>(`projects/${projectId}?${qs.stringify(sudo)}`, true);
+      project = await this.fetchJson<Project>(
+        `projects/${projectId}?${qs.stringify(sudo)}`,
+        true,
```

Comment:
This is actually pretty surprising. After looking at what's happening here, I found: https://github.com/Microsoft/TypeScript/issues/7485. So TS 2.4 would've caught this. Anyway, fixed.